app/models/group.rb

```diff
@@ -1,4 +1,9 @@
 class Group < ApplicationRecord
   has_many :user_groups
   has_many :users, through: :user_groups
+  has_many :lists
+  belongs_to :user, foreign_key: :owner_id
```

Comment:
What do you think of accessing a group's owner by being able to call `group.owner`? I think this would be less confusing than `user` since a group will have a `users` relationship. If you agree with the change, you can do this by replacing line 5 with `belongs_to :own, foreign_key: :owner_id, class_name: 'User`.

Reply:
I agree with this change, it makes it much more clear. It is implemented now.